.storybook/preview.tsx

```diff
@@ -19,12 +19,12 @@ const preview: Preview = {
       },
     },
   },
-  argTypes: {
-    onBlur: { action: 'onBlur' },
-    onChange: { action: 'onChange' },
-    onFocus: { action: 'onFocus' },
-    onInputChange: { action: 'onInputChange' },
-  },
+  // argTypes: {
+  //   onBlur: { action: 'onBlur' },
+  //   onChange: { action: 'onChange' },
+  //   onFocus: { action: 'onFocus' },
+  //   onInputChange: { action: 'onInputChange' },
+  // },
```

Comment:
This will be revisited in a follow-up PR.
Currently we were adding these types to all components stories, which is not expected.